### ISSUE TITLE
[ECS] host.name field not forced to lowercase (Recommended via ECS)

### ISSUE THREAD

**Issue by nicpenning** [elastic/endpoint-package#453]
It appears that alerts and other information generated by the Elastic Defend integration for endpoints show the host name in it's original casing. For example, an Alert generated by Elastic defend (in 8.10.4) could show `host.name:DESKTOP-75.org.local`

Will it be possible to lowercase the host.name field as recommended in the ECS docs? 
https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-name

**Comment by ferullo** [comment]
I actually just spent some time trying to cause this behavior and was unable. Endpoint always downcases `host.name`. Are you sure you're seeing this issue in documents with `event.module : endpoint`? Is it possible you have something in Logstash or a custom pipeline that may be altering the `host.name`? Do you see this on all hosts or just Windows (and if just Windows, all Windows or just some Windows)?

**Comment by nicpenning** [comment]
Good point, let me check for you!

**Comment by nicpenning** [comment]
From the data stream: logs-endpoint.alerts-*

Looks like I was mistaken - This appears to be an outdated agent (8.8.1)
```JSON
    "agent": {
      "build": {
        "original": "version: 8.8.1, compiled: Sat Jun 3 02:00:00 2023, branch: 8.8, commit: a4a4ae26ff6a2e57b107baa1c8d49f43fe1cd77c"
      },
      "id": "redacted",
      "type": "endpoint",
      "version": "8.8.1"
```

```JSON
 "event": {
      "severity": 99,
      "code": "behavior",
      "risk_score": 99,
      "created": "2023-11-20T15:51:59.9505896Z",
      "kind": "alert",
      "module": "endpoint",
      "type": [
        "info",
        "allowed"
      ],
      "agent_id_status": "verified",
      "sequence": 14966,
      "ingested": "2023-11-20T15:52:00Z",
      "action": "rule_detection",
      "id": "NJi++xAhvFZ2Unsn+++++Eow",
      "category": [
        "malware",
        "intrusion_detection"
      ],
      "dataset": "endpoint.alerts",
      "outcome": "success"
    },
```

If this was resolved past 8.8.1, this can be closed. Sorry to waste your time!

**Comment by nicpenning** [comment]
I can clearly see any agents beyond 8.8.1 do not have this issue. Closing!

**Comment by ferullo** [comment]
Ah, yes, you are right. It looks like it was fixed in 8.9.
